src/App.tsx

```diff
@@ -129,75 +143,46 @@ export const App: React.FC = () => {
               ×
             </button>
 
-            {/* 'is-active' class puts this modal on top of the todo */}
             <div data-cy="TodoLoader" className="modal overlay is-active">
               <div className="modal-background has-background-white-ter" />
               <div className="loader" />
             </div>
-          </div>
-        </section>
-
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
-
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
-
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
-
-            <a
-              href="#/completed"
-              className="filter__link"
-              data-cy="FilterLinkCompleted"
-            >
-              Completed
-            </a>
-          </nav>
-
-          {/* this button should be disabled if there are no completed todos */}
-          <button
-            type="button"
-            className="todoapp__clear-completed"
-            data-cy="ClearCompletedButton"
-          >
-            Clear completed
-          </button>
-        </footer>
+          </div> */}
+
+        {!!todos.length && (
+          <Footer
+            onSetFilter={handlerSetFilter}
+            countOfTodos={counterTodos}
+            selectedFilter={filterBy}
+          ></Footer>
```

Comment:
```suggestion
          <Footer
            onSetFilter={handlerSetFilter}
            countOfTodos={counterTodos}
            selectedFilter={filterBy}
          / >
```
You don't need the close tag if you don't pass `children`